bool get_pose
---
float64 x
float64 y
float64 z
float64 rx
float64 rz
